types/finalhandler/index.d.ts

```diff
@@ -1,20 +1,21 @@
-// Type definitions for finalhandler
+// Type definitions for finalhandler v1.1.1
```

Comment:
This appears to be a malformed header. Could you remove the line in the `tslint.json` file in this folder that says `dt-header: false` and fix any errors that come up? You will want this first line to say `Type definitions for finalhandler 1.1`. You are welcome to remove any other such line from the `tslint.json` file, though that might be a lot of work depending on what issues are being ignored. Thanks!

Reply:
I deleted all the `rules` from `tslint` and fixed all the linting problems, should be good now.